src/components/PreviewCard.jsx

```diff
@@ -17,28 +17,30 @@ const randomIntFromInterval = (min, max) => {
 };
 
 const randomImage = () => {
-  const imageArray = [repocube1,repocube2,repocube3,repocube4];
-  return imageArray[randomIntFromInterval(0,3)];
+  const imageArray = [repocube1, repocube2, repocube3, repocube4];
+  return imageArray[randomIntFromInterval(0, 3)];
 };
 
 const useStyles = makeStyles(() => ({
   card: {
-      marginBottom: 2,
-      display:"flex",
-      flexDirection:"row",
-      alignItems:"center",
-      background:"#FFFFFF",
-      boxShadow:"0px 5px 10px rgba(131, 131, 131, 0.08)",
-      borderRadius:"24px",
-      flex:"none",
-      alignSelf:"stretch",
-      flexGrow:0,
-      order:0,
-      width:"100%",
+    marginBottom: 2,
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    background: "#FFFFFF",
+    boxShadow: "0px 5px 10px rgba(131, 131, 131, 0.08)",
+    borderRadius: "24px",
```

Comment:
try to use relative units, i.e. em/rem or %

Reply:
changed all px measurements to rem